packages/plugins/experimental/plugin-wnfs/src/load.ts

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright 2024 DXOS.org
+//
+
+import type { Blockstore } from 'interface-blockstore';
+import { CID } from 'multiformats';
+import * as Uint8Arrays from 'uint8arrays';
+import { AccessKey, PrivateForest, PrivateDirectory, PrivateNode } from 'wnfs';
+
+import { type Space } from '@dxos/react-client/echo';
+
+import { Rng, store } from './common';
+
+// 🚀
```

Comment:
prefer descriptive comments, I'm not sure this communicates intent clearly enough